app/components/how-do-you-feel.jsx

```diff
@@ -0,0 +1,95 @@
+//https://github.com/EnCiv/civil-pursuit/issues/62
+import React, { useState } from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+
+function Howdoyoufeel(props) {
+    const {
+        className = '',
+        onDone = () => { },
+        title = '',
+        disabled = false,
+        disableOnClick = false,
+        children,
+        ...otherProps
+    } = props
+    const [isDisabled, setIsDisabled] = useState(disabled)
+    const classes = buttonStyles()
+    const combinedClassName = cx(classes.buttonBase, className)
+
+    return (
+        <div>
+            <p style={{ fontSize: "35px", fontWeight: "normal" }}>How do you feel about it</p>
```

Comment:
Must use rem, not px.  Px is not responseive, the number of px in a rem will be different on different device.  Why not use a className here?